forgerock-openbanking-config/forgerock-openbanking-git-config/aspsp/rs/rs-store/rs-store.yml

```diff
@@ -12,6 +12,19 @@ server:
 
 rs:
   data.internal_ids.show: true
+  # Refer to https://github.com/OpenBankingToolkit/openbanking-reference-implementation/issues/344
+  data.internal_ids.direct_debit.show: false
+  data.internal_ids.offer.show: false
+  data.internal_ids.product.show: false
+  data.internal_ids.statement.show: true
+  data.internal_ids.domestic.transaction.show: true
+  data.internal_ids.international.transaction.show: true
+  data.internal_ids.domestic.beneficiary.show: false
+  data.internal_ids.international.beneficiary.show: true
+  data.internal_ids.domestic.scheduled_payment.show: false
+  data.internal_ids.international.scheduled_payment.show: false
+  data.internal_ids.domestic.standing_order.show: false
+  data.internal_ids.international.standing_order.show: false
```

Comment:
@BohoCode - I've also set these values here: https://github.com/ForgeCloud/ob-sandbox-config/pull/20

I'm a little uneasy about having customer specific values here - let's discuss when you're free.

Reply:
I agree. These should all be true and then overridden in specific customer config when needed.

Reply:
As discussed, `data.internal_ids.show` toggles the behaviour between all of them being displayed and none of them. I've set the new values to false, as otherwise setting `data.internal_ids.show` to false won't have any effect.